src/openforms/authentication/api/views.py

```diff
@@ -39,6 +40,25 @@ def get_objects(self):
         return list(register.iter_enabled_plugins())
 
 
+@extend_schema_view(
+    get=extend_schema(summary=_("List available Yivi attribute groups")),
+)
+class AttributeGroupListView(ListMixin, APIView):
```

Comment:
this looks like it could/should be a plain `rest_framework.generics.ListApiView` instead of using our own `ListMixin` alternative. The latter is inspired by the former do deal with non-database data (this is described in the `ListMixin` docstring 😉 )